#! /bin/sh /usr/share/dpatch/dpatch-run
## fix_resubscribe.dpatch by  <mikma@users.sourceforge.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix handling of Re-SUBSCRIBE in eventserver.
## DP: Remove parameters in call to PackageM:is_allowed_subscribe.

@DPATCH@
diff -urNad yxa-1.0+20090326~/src/event/subscription.erl yxa-1.0+20090326/src/event/subscription.erl
--- yxa-1.0+20090326~/src/event/subscription.erl	2009-03-25 09:17:49.000000000 +0100
+++ yxa-1.0+20090326/src/event/subscription.erl	2009-07-15 22:11:55.000000000 +0200
@@ -754,7 +754,7 @@
 		       "~p is allowed to re-subscribe to presentity ~p",
 		       [PackageS, PackageM, Subscriber, Presentity]),
 	    case PackageM:is_allowed_subscribe(PackageS, SubNum + 1, NewRequest, YxaCtx,
-					       LogTag, THandler, Subscriber, Presentity,
+					       Subscriber, Presentity,
 					       PkgState) of
 		{ok, SubState1, Status, Reason, ExtraHeaders, Body, NewPkgState}
 		when SubState1 == active orelse SubState1 == pending orelse SubState1 == terminated,
